src/components/TriggersDataTable.vue

```diff
@@ -0,0 +1,139 @@
+<template>
+  <el-table
+    :data="tableData"
+    :default-sort="{ prop: 'date', order: 'descending' }"
+  >
+    <el-table-column
+      v-for="col in tableColumns"
+      :key="col.id"
+      :prop="col.prop"
+      :label="col.label"
+      :sortable="col.isSortable"
+      :column-key="col.prop"
+      :width="col.width"
+    />
+    <el-table-column prop="action" label="">
+      <div class="action-icons">
+        <i class="fa-solid fa-clipboard-check icon"></i>
+        <i class="fa-solid fa-bell icon"></i>
+      </div>
+    </el-table-column>
+  </el-table>
+  <el-pagination layout="prev, pager, next" :total="50"> </el-pagination>
+</template>
+
+<script>
+import { ElTable, ElTableColumn, ElPagination } from 'element-plus';
+import 'element-plus/es/components/table/style/css';
+import 'element-plus/es/components/table-column/style/css';
+import 'element-plus/es/components/pagination/style/css';
```

Comment:
This code is repeated in all three table components, so it could be extracted into the scss file you're already importing